docs/dictionary/command/filter.lcdoc

```diff
@@ -108,22 +113,22 @@ You can match instances of special chars as follows:
 
 
 For example, the wildcard <filterPattern> `[[]A]*` will match any
-string beginning with `[A]`. Broken down, there is '[[]' which equates
-to an open square bracket '[' followed by 'A]*' as the closing square
+string beginning with `[A]`. Broken down, there is `[[]` which equates
+to an open square bracket `[` followed by `A]*` as the closing square
 bracket is not a special character.
 
 The three bracketed forms can be combined to create more complex
 character classes, for example the pattern [!abcA-C] matches any
 character which is not a, b or c (upper or lower case)
 
-If 'regex pattern' is specified, the <filterPattern> evaluates to a
-regular expression.
-
-If no <filterPattern> type is specified, the <filterPattern> is handled
-as a wildcard.
+>**Note:** If the <paternType> `regex pattern` is specified, 
```

Comment:
@seaniepie the `<>` should only be used for parameters or doc referenced docs. I'm thinking it's better to make just the bit about the no type specified the note. That's the bit that is easy to miss.

Reply:
@montegoulding I think your looking at an older commit. Jump to the latest and mark which ones you think are still valid. Particularly the one about having wildcard and regex as parameters. Are they really parameters as they are more like keywords than a placeholder for a value.